src/mpi/romio/adio/include/adioi.h

```diff
@@ -225,6 +225,8 @@ struct ADIOI_Fns_struct {
                                          MPI_Datatype datatype, int file_ptr_type,
                                          ADIO_Offset offset, ADIO_Request * request,
                                          int *error_code);
+    void (*ADIOI_xxx_SetLock) (ADIO_File fd, int cmd, int type, ADIO_Offset offset, int whence,
```

Comment:
@reflectored this should be `int` return value, right?

Reply:
I can fix before merging, just looking to confirm.

Reply:
Correct, sorry I wanted to go over the code again to make sure. I see the return value for this function is not handled throughout the source code, but that will be probably left for another change.